### PR TITLE
brigade org playbook

### LINKED ISSUE #965: Create project card: Brigade Organizers Playbook (BOP)
### Overview
Provide collateral for the HackforLA website

### Action Items
- [x]   Gather items
  - [x]  600 x 400 image
     - [x]  Alt image text
  - [x] 1500 x 700 hero image (please do not put project title on hero image)
     - [x]  Alt image text
  - [x]  Name of project
  - [x]  A blurb about your project
  - [x]  Links (github, slack channel url, Testing Site, Live Site, etc.)
  - [x]  Any resources for a Getting Started link (a link to a wiki, readme(s) or both)
  - [x]  What you are looking for skills wise
  - [x]  Partner(s)
  - [x]  Location
  - [x]  Status
- [x]   Add project Card to website

### Resources/Instructions
See project cards on the hackforla.org website for examples

### PR DESCRIPTION
**Pull request by matthewlee626**
Fixes #965

### CODE REVIEW

File: _projects/brigade-organizers-playbook.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+identification: '336742445'
+title: Brigade Organizers Playbook
+description: Each of the 80+ Code for America Brigades is an experiment which generates valuable learnings and new effective processes and practices. However, Brigades and other civic tech volunteer organizations need not start from scratch. This project aims to improve existing structures and create new ones that make it easier to share replicable processes and practices so that organizers and members can iterate on each others work, improving outcomes for the whole network.
+image: /assets/images/projects/brigade-organizers-playbook.png
+alt: 'Brigade Organizer's Playbook Extension Project'
```
Comment (drubgrubby) [inline _projects/brigade-organizers-playbook.md]: (This has been edited from an earlier version.  Bonnie suggested a diffefent fix.)

As you stated in the issue, this page is throwing a YAML exception. 

The problem seems to be the apostrophe in line 6 - alt: 'Brigade Organizer's Playbook Extension Project'. It's trying to process the apostrophe within the string as part of the code and throwing a YAML Exception.

You can solve this by using double quotes outside the string: 
```
The sentence would look like: alt: "Brigade Organizer's Playbook Extension Project"
```

**Review by drubgrubby — CHANGES_REQUESTED** [review]
I haven't done a close inspection yet, but this looks good on first glance. 

Once you fix the YAML exception you can push the changes and make a note here and in the issue that the changes have been fixed and we can check it and merge it. 

In general, it's best not to create a pull request until you have completely solved the problem and you think the issue is ready to merge.   Until then, you should just keep the comments and questions on the issue itself.

**Comment by matthewlee626** [comment]
Sorry about that! I'll keep that in mind for future issues.￼

**Comment by matthewlee626** [comment]
Addressed the problem in the issue! This should be good to merge.

**Pull request merged**